.github/workflows/schedule-daily.yml

```diff
@@ -136,11 +136,10 @@ jobs:
           BAZEL_COMMAND: "test"
           BAZEL_TARGETS: "//rs/rosetta-api/..."
           BAZEL_CI_CONFIG: "--config=ci --repository_cache=/cache/bazel"
-          BAZEL_EXTRA_ARGS: "--keep_going --test_tag_filters=fi_tests_nightly --test_env=SSH_AUTH_SOCK"
+          BAZEL_EXTRA_ARGS: "--keep_going --test_tag_filters=fi_tests_nightly --test_env=SSH_AUTH_SOCK --test_timeout=7200"
```

Comment:
Is the `test_timeout` related? what is it for?

Reply:
The `bazel` `--test_timeout` is needed for the FI tests - after first fixing the double `if`, the job still failed since one of the tests took over an hour to complete.